tests/test_tracking_lattice_cmfd/test_tracking_lattice_cmfd.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+
+import os
+import sys
+import math
+from collections import OrderedDict
+sys.path.insert(0, os.pardir)
+sys.path.insert(0, os.path.join(os.pardir, 'openmoc'))
+from testing_harness import TrackingTestHarness
+from input_set import LatticeGridInput
+
+import openmoc
+
+class TrackingLatticeGridCMFDTestHarness(TrackingTestHarness):
```

Comment:
ditto, what does `LatticeGrid` mean beyond just `Lattice`? Unless "Grid" actually means something, I think it should be removed to eliminate any source of confusion.